### PR TITLE
Org.codeberg.dimkard.glossaico

### PR DESCRIPTION
**Pull request by dimkard**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributable by any party.
- [x] I am an upstream contributor to the project. 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.codeberg.dimkard.glossaico.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "id": "org.codeberg.dimkard.glossaico",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
```
Comment (barthalion) [inline org.codeberg.dimkard.glossaico.json:4]:
```suggestion
    "runtime-version": "5.15-21.08",
```

File: org.codeberg.dimkard.glossaico.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "glossaico",
+    "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```
Comment (barthalion) [inline org.codeberg.dimkard.glossaico.json:9]:
```suggestion
    "--socket=fallback-x11",
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Glossaico
```
Comment (barthalion) [inline README.md:1]: FYI it's highly unlikely any user will ever see this. It's better to include anything user-facing in appdata/metainfo.

**Comment by barthalion** [comment]
bot, build org.codeberg.dimkard.glossaico

File: org.codeberg.dimkard.glossaico.json (hunk 1 above)
Comment (dimkard) [reply org.codeberg.dimkard.glossaico.json:4]: I tried this first, but pyside2 did not build for me, IIRC due to something related to clang.

File: README.md (hunk 3 above)
Comment (dimkard) [reply README.md:1]: So, you suggest that I remove the README file?

**Comment by nedrichards** [comment]
bot, build org.codeberg.dimkard.glossaico

**Pull request closed without merging**